modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewChangesDisplayContext.java

```diff
@@ -1374,7 +1374,12 @@ private JSONArray _getDropdownItemsJSONArray(
 
 		JSONArray jsonArray = JSONFactoryUtil.createJSONArray();
 
-		if (FeatureFlagManagerUtil.isEnabled("LPS-179035")) {
+		if (FeatureFlagManagerUtil.isEnabled("LPS-179035") &&
+			StringUtil.equals(
+				ParamUtil.get(
+					_renderRequest, "mvcRenderCommandName", StringPool.BLANK),
```

Comment:
Let's use `ParamUtil.getString` to get the parameter. Also the default defaultValue for String is the blank string, so unless you want a different default value, we can omit the last param.